airgun/views/cloud_insights.py

```diff
@@ -0,0 +1,97 @@
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly4 import Button
+from widgetastic_patternfly4 import Pagination
+from widgetastic_patternfly4.ouia import Modal
+from widgetastic_patternfly4.ouia import PatternflyTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import InventoryBootstrapSwitch
+
+
+class CloudTokenView(BaseLoggedInView):
+    """RH Cloud Insights Landing page for adding RH Cloud Token."""
+
+    rhcloud_token = TextInput(locator='//input[contains(@aria-label, "input-cloud-token")]')
+    save_token = Button(locator='//button[text()="Save setting and sync recommendations"]')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.rhcloud_token, exception=False) is not None
+
+
+class RemediationView(Modal):
+    """ Remediation window view"""
+
+    remediate = Button(locator='//button[text()="Remediate" and @type="submit"]')
+    cancel = Button(locator='//button[text()="Cancel"]')
+    table = PatternflyTable(
+        component_id='OUIA-Generated-Table-2',
+        column_widgets={
+            'Hostname': Text('./a'),
+            'Recommendation': Text('./a'),
+            'Resolution': Text('.//a'),
+            'Reboot Required': Text('.//a'),
+        },
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.title, exception=False) is not None
+
+
+class CloudInsightsView(BaseLoggedInView, SearchableViewMixin):
+    """Main RH Cloud Insights view."""
+
+    title = Text('//h1[text()="Red Hat Insights"]')
+    insights_sync_switcher = InventoryBootstrapSwitch(class_name='insights_sync_switcher')
+    start_hits_sync = Button(locator='//button[text()="Start recommendations sync"]')
```

Comment:
Same recommendations to use `Button` class locator construction and just define these as `Button('Start recommendations sync')` and `Button('Remediate')`